kotlin-vercel-ncc/src/jsMain/kotlin/vercel/ncc.kt

```diff
@@ -0,0 +1,12 @@
+package vercel
+
+import kotlinx.coroutines.await
+import kotlin.js.Promise
+
+@JsModule("@vercel/ncc")
+@JsNonModule
+@JsName("ncc")
```

Comment:
Does it have effect?
Looks like you import module and name isn't used :(

Reply:
Just copy & paste problem and not knowing what I'm doing, copying a similar setup, but there the `JsModule` was not on the function.